### PR TITLE
consoole.log

### PR DESCRIPTION
**Pull request by michael-grace**
not removing it, just commenting it

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,7 +40,7 @@ <h1>no</h1>
 <script>
     const isToday = () => {
         // 12 feb 2020
-        console.log(1)
+        // console.log(1)
```
Comment (markspolakovs) [inline index.html:43]: no

**Review by markspolakovs — CHANGES_REQUESTED** [review]
no

File: index.html (hunk 1 above)
Comment (michael-grace) [reply index.html:43]: // no

Comment (ColinRoitt) [reply index.html:43]: There has been a major update on this feature as of this post - I appreciate the work you have put into this and can't imagine it's a big job to work this feature into the newer version and would urge you to do so over the coming weeks.

**Comment by ColinRoitt** [comment]
As of today, there has been a significant update to this feature - we appreciate your work and would urge you to port this to a newer version, If you wish to please submit a new PR with this feature in a branch ahead of master - thanks, Colin

**Comment by michael-grace** [comment]
> As of today, there has been a significant update to this feature - we appreciate your work and would urge you to port this to a newer version, If you wish to please submit a new PR with this feature in a branch ahead of master - thanks, Colin

I hadn't even noticed this message 😂😂

**Pull request closed without merging**